x/mongo/driver/session/server_session_test.go

```diff
@@ -11,20 +11,38 @@ import (
 	"time"
 
 	"github.com/stretchr/testify/require"
+	"go.mongodb.org/mongo-driver/internal/testutil/assert"
+	"go.mongodb.org/mongo-driver/mongo/description"
 )
 
 func TestServerSession(t *testing.T) {
-
 	t.Run("Expired", func(t *testing.T) {
-		sess, err := newServerSession()
-		require.Nil(t, err, "Unexpected error")
-		if !sess.expired(0) {
-			t.Errorf("session should be expired")
-		}
-		sess.LastUsed = time.Now().Add(-30 * time.Minute)
-		if !sess.expired(30) {
-			t.Errorf("session should be expired")
-		}
+		t.Run("non-lb mode", func(t *testing.T) {
+			sess, err := newServerSession()
+			require.Nil(t, err, "Unexpected error")
```

Comment:
[nit] Can you use `assert` instead of `require` for consistency with `lb mode` test? And also fix the error messages to be like the ones in `lb mode`?

Reply:
Done.